app_waitMysql.js

```diff
@@ -5,6 +5,16 @@
 const path = require("path");
 
 const Mysql = require("mysql"); // our  {DB Connection}
+const Sentry = require("@sentry/node");
+const { version } = require("./package");
+Sentry.init({
+   dsn: "https://0363fe8e68b2fbd38a807305a3a1212c@o144358.ingest.sentry.io/4505945407488000",
```

Comment:
Should this be passed in using our ENV variables?  
```
const AB = require("@digiserve/ab-utils");
const env = AB.defaults.env;

... 
Sentry.init({
dsn: env("SENTRY_DSN"),

```

Should we have the option for other implementations to have separate Sentry configs than us.